### PR TITLE
Deploy with argocd

### PR DESCRIPTION
**Pull request by garryod**
Depends on #165

### CODE REVIEW

**Review by DiamondJoseph — APPROVED** [review]
Seems reasonable. @gilesknap might have insight into the Argo-ness? We need to get more familiar

File: charts/apps/values.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+project: ""
+
+destination:
+  name: ""
+  server: ""
+  namespace: ""
```
Comment (tpoliaw) [inline charts/apps/values.yaml:6]: Why are these not required?

File: charts/apps/values.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+bundler:
+  enabled: true
+  repoUrl: https://github.com/DiamondLightSource/authz.git
+  targetRevision: HEAD
```
Comment (tpoliaw) [inline charts/apps/values.yaml:11]: Is this deploying every commit?

**Review by tpoliaw — COMMENTED** [review]
Based entirely on guessing how argocd might work, but looks believable.

**Pull request merged**